src/DTO/Bandit/AttributeSet.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Eppo\DTO\Bandit;
+
+class AttributeSet
+{
+    /**
+     * Types to automatically classify as Categorical
+     */
+    private const CATEGORICAL_TYPES = ["boolean", "string"];
+
+    /**
+     * Types to automatically classify as Numeric
+     * `float` and `double` are the sane datatype in php.
+     * In PHP, for historical reasons, `gettype` returns `double`.
```

Comment:
I appreciate the comment--I was just wondering if the two were different somewhow!